tests/extmod/ssl_threads.py

```diff
@@ -0,0 +1,58 @@
+# Ensure that SSL sockets can be allocated from multiple
```

Comment:
Since this test uses the `tls` module (and not `ssl`) I suggest calling it `tls_threads.py`.